src/app/containers/Ad/index.jsx

```diff
@@ -3,24 +3,19 @@ import pathOr from 'ramda/src/pathOr';
 import { RequestContext } from '../../contexts/RequestContext';
 import { ServiceContext } from '../../contexts/ServiceContext';
 import Amp from './Amp';
-import useToggle from '#hooks/useToggle';
+import Canonical from './Canonical';
 
 const AdContainer = () => {
   const { isAmp } = useContext(RequestContext);
   const { service, ads } = useContext(ServiceContext);
   const hasAds = pathOr(false, ['hasAds'], ads);
```

Comment:
I agree that this is a bit confusing, but I think we still want to check the ad toggle value here before rendering the component, whether it is AMP or Canonical. We have `ads` set to `false` on the live environment in the toggles file in Simorgh, and also have `enableFetchingToggles` set to `false`. This ensures that we do not show ads on the live environment until they are ready. If we remove this check here, I think the unfinished AMP ad component is going to start rendering on live?

Reply:
Some thoughts, been discussing it with @thekp and I think these are right

- If we implement the toggles logic in AMP it can't do the regular toggles fetch. So if we default it to `false` then the request can't run and it can never be set to `true`.
- amp-access is doing this in AMP, but it isn't reading the `defaultToggles`.
- From what I'm seeing on Canonical at the moment, if we default ads to enabled, but the remote turns them off, there's a brief moment where you see the ad, and then it gets toggled off. It's a little bit jarring, and would also show for UK users. This is solved by disabling by default in the toggles config, which I think they're supposed to be anyway?

Reply:
> if we remove this check here, I think the unfinished AMP ad component is going to start rendering on live?

We can just toggle it off on LIVE iSite, as it is per application. e.g 
![image](https://user-images.githubusercontent.com/30599794/80611494-928e0400-8a32-11ea-9520-d6a921207d84.png)

Reply:
I think it would be really good to have another refinement, based on the outcome of our session tomorrow, and document at a high level everything that could happen here - we'll need tickets to make sure that the forms are set up and working correctly, and that we have production ready logging etc in the toggles service so we can keep an eye on how it is performing. Maybe for now we need to decouple the toggles for 'should this feature be in production' and the 'should this user be seeing ads', so we can be sure that we don't release anything that's unfinished, but be able to implement the client side toggles work in the way we think is correct?

Reply:
As this is back in code review - I still think we should ensure this whole component is toggled off at the root level while this is in development, as we don't want to accidentally show any in progress ads work to any users

Reply:
There are in progress conversations around the role of the toggles service regarding geoIP lookup too, so it may be safer to just use the hard coded toggles at the moment, and then implement anything on top of that once those conversations are resolved?

Reply:
@tochwill [Toggles config](https://github.com/bbc/simorgh/blob/latest/src/app/lib/config/toggles/index.js) shows that `ads` are toggled ON on local/test and OFF on live. Is this not sufficient?

Reply:
@sareh, yeah that is sufficient, but this PR is removing the check to see if ads are enabled (on line 12), and always rendering the Ad component. The canonical container is checking the toggles before rendering anything, but now that this check is removed, the AMP ad is always going to be rendered, including in live, and will make a call to the remote toggles service. I'm saying we should keep this level of check to ensure we don't render the AMP ad on live.

Reply:
@tochwill Thanks for clarifying. I missed that. Yes, I'd prefer to keep the check in the `/Ad/index.jsx` rather than move it to `/Ad/Canonical` `/Ad/Amp`.

Reply:
I can move the toggle check into that file, but I believe it needs to be within the `!isAmp` logic. The useToggles fetch doesn't work within Amp so if the default is set to disabled the call wouldn't occur and it couldn't be set enabled.

Everytime I write this it sounds confusing!

Reply:
But currently its true that the toggles defaults will stop it from being rendered on live, so I can add it back in until an amp solution is ready.

Reply:
Yeah, you're right that we will need the toggles call for AMP pages to come from the client, to check to see if that service has ads toggled on. We're using the default toggle from the `json` file in Simorgh to make sure ads are blanket turned off in live, and we have disabled the client side toggles fetch in live for canonical also. Going forward, we need to remove the client fetch for canonical, and fetch this information on the server instead. It's got into a bit of a confusing state, so we should document/diagram this behaviour.